en/user_guide/content.rst

```diff
@@ -0,0 +1,62 @@
+Contents
+========
+
+Contens are instances of a Content Types. A Content Type is the pattern of a family of Contents. If
+"Car" was a Content Type, "Ford mustang" or "Dodge Viper" could be contents of that type.
+See `Content Types`_ documentation for more information.
+
+
+Listing
+-------
+Contents listing is accessed in the Editorial section, by clicking on the "Contents" link. A submenu
+lists all Content Types availables.
+
+.. image:: ../../images/content_list_submenu.png
+
+By clicking a Content Type, the list of related contents appears.
+
+.. image:: ../../images/content_list.png
+
+This list has a pagination and filters, and shows the name, the status, the version and the language
+of existing contents. From this list, new content can be created, and existing ones can be edited or
+deleted.
+
+
+Creation
+--------
+From the content list, creation form can be accessed by clicking on the "Add" button. Information asked
```

Comment:
I would be more direct by saying ``By clicking on the "Add" button, you access the creation form. The form will be related on the content type previously selected``